src/main/kotlin/org/jetbrains/bio/io/Bed.kt

```diff
@@ -305,19 +305,34 @@ data class BedFormat(
 /**
  * Reads [BedEntry]-s from the supplied [reader].
  *
- * Set [stringency] to your preferred mode before starting to parse. See [Stringency] for mode explanation.
+ * [BedParser] is usually used via [BedFormat.parse] method. It implements [Iterable] over [BedEntry],
+ * so it's normally a receiver of [map] or [forEach] calls:
+ *      bedFormat.parse(file) { parser ->
+ *          parser.forEach { bedEntry -> println(bedEntry) }
+ *      }
  *
- * Use [linesFailedToParse] property to find out how many lines were skipped due to parsing errors.
+ * @param reader The reader that provides lines to convert into [BedEntry]-s.
```

Comment:
Seems should be `@property` according to KDoc documentation (see example at https://kotlinlang.org/docs/reference/kotlin-doc.html)

Reply:
Done.